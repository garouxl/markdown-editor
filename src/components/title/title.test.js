/* 'use strict'

import { expect } from 'chai'
import Title from './index'

test('Title should be a function', () => {
  expect(Title('Teste')).to.be.a('function')
})
 */
