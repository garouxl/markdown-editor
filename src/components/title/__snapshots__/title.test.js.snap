exports[`Should Title match to snapshot 1`] = `
<h1>
  TÃ­tulo
</h1>
`;
